src/uniparc/components/entry/Entry.tsx

```diff
@@ -49,7 +49,7 @@ const Entry: FC = () => {
 
   // if URL doesn't finish with "entry" redirect to /entry by default
   useEffect(() => {
-    if (match && !match.params.subPage) {
+    if (match && !match.params.subPage && accession) {
```

Comment:
This is not really part of the PR, so more of a generic comment: Should we have this redirect logic on all the entry page? This way, it's more consitent throughout the website (even if we don't end up having tabs on all the kinds of entry pages).

Reply:
Yes, or at least pages which use tabs...should it be a custom hook then?

Reply:
It could be a custom hook, yes.
But yeah, my point is should all entries be /<id>/entry anyway, to make it consistent?

Reply:
I see... I don't think so, as we will still have to provide paths to the entry without that suffix

Reply:
This is doing redirect anyway for those entries with tabs, so it's the same. The non-`/entry` version would still work for all of them

Reply:
Same logic in BLAST and Align results too.

Reply:
Ok - I think this could be tied in with the tabs too so would require more thought. I'll create a Jira